bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -117,14 +117,33 @@ void DeleteInstance(bigtable::InstanceAdmin instance_admin, int argc,
 //! [list clusters]
 void ListClusters(bigtable::InstanceAdmin instance_admin, int argc,
                   char* argv[]) {
-  auto cluster_list = instance_admin.ListClusters();
+  if (argc != 2) {
+    throw Usage{"list-clusters <project-id> <instance-id>"};
+  }
+
+  auto cluster_list = instance_admin.ListClusters(ConsumeArg(argc, argv));
   std::cout << "Cluster Name List" << std::endl;
   for (auto const& cluster : cluster_list) {
     std::cout << "Cluster Name:" << cluster.name() << std::endl;
   }
 }
 //! [list clusters]
 
+//! [list all clusters]
```

Comment:
In a future PR we need to implement the `ListClusters` function in the instance admin emulator and then we need to call these new examples for `run_examples_utils.sh`.  Can you make a note in #490 to that effect or create a new bug for it?